### ISSUE TITLE
Instructions on how to restore a healthy state after node reboot

### ISSUE THREAD

**Issue by wibed** [openebs/mayastor#1182]
given a node reboot etcd pod 2 & 3 are stuck in a reboot loop

how do i reinstate a healthy state?

my pools error out after restart
```
NAMESPACE   NAME        NODE               STATUS   CAPACITY   USED   AVAILABLE
mayastor    pool-on-1   talos-10-0-48-44   Error    0          0      0
mayastor    pool-on-2   talos-10-0-48-45   Error    0          0      0
mayastor    pool-on-3   talos-10-0-48-46   Error    0          0      0
```

**Comment by wibed** [comment]
logs

```
2022-08-21 12:42:47.411204 I | pkg/flags: recognized and used environment variable ETCD_ADVERTISE_CLIENT_URLS=http://mayastor-etcd-0.mayastor-etcd-headless.mayastor.svc.cluster.local:2379
2022-08-21 12:42:47.411285 I | pkg/flags: recognized and used environment variable ETCD_AUTO_TLS=false
2022-08-21 12:42:47.411297 I | pkg/flags: recognized and used environment variable ETCD_CLIENT_CERT_AUTH=false
2022-08-21 12:42:47.411306 I | pkg/flags: recognized and used environment variable ETCD_DATA_DIR=/bitnami/etcd/data
2022-08-21 12:42:47.411333 I | pkg/flags: recognized and used environment variable ETCD_INITIAL_ADVERTISE_PEER_URLS=http://mayastor-etcd-0.mayastor-etcd-headless.mayastor.svc.cluster.local:2380
2022-08-21 12:42:47.411349 I | pkg/flags: recognized and used environment variable ETCD_INITIAL_CLUSTER=mayastor-etcd-0=http://mayastor-etcd-0.mayastor-etcd-headless.mayastor.svc.cluster.local:2380,mayastor-etcd-1=http://mayastor-etcd-1.mayastor-etcd-headless.mayastor.svc.cluster.local:2380,mayastor-etcd-2=http://mayastor-etcd-2.mayastor-etcd-headless.mayastor.svc.cluster.local:2380
2022-08-21 12:42:47.411367 I | pkg/flags: recognized and used environment variable ETCD_INITIAL_CLUSTER_STATE=new
2022-08-21 12:42:47.411374 I | pkg/flags: recognized and used environment variable ETCD_INITIAL_CLUSTER_TOKEN=etcd-cluster-k8s
2022-08-21 12:42:47.411405 I | pkg/flags: recognized and used environment variable ETCD_LISTEN_CLIENT_URLS=http://0.0.0.0:2379
2022-08-21 12:42:47.411430 I | pkg/flags: recognized and used environment variable ETCD_LISTEN_PEER_URLS=http://0.0.0.0:2380
2022-08-21 12:42:47.411438 I | pkg/flags: recognized and used environment variable ETCD_LOG_LEVEL=info
2022-08-21 12:42:47.411463 I | pkg/flags: recognized and used environment variable ETCD_NAME=mayastor-etcd-0
2022-08-21 12:42:47.411471 I | pkg/flags: recognized and used environment variable ETCD_PEER_AUTO_TLS=false
2022-08-21 12:42:47.411503 W | pkg/flags: unrecognized environment variable ETCD_TRUSTED_CA_FILE=
2022-08-21 12:42:47.411521 W | pkg/flags: unrecognized environment variable ETCD_ON_K8S=yes
2022-08-21 12:42:47.411526 W | pkg/flags: unrecognized environment variable ETCD_SNAPSHOTS_DIR=/snapshots
2022-08-21 12:42:47.411531 W | pkg/flags: unrecognized environment variable ETCD_BIN_DIR=/opt/bitnami/etcd/sbin
2022-08-21 12:42:47.411535 W | pkg/flags: unrecognized environment variable ETCD_VOLUME_DIR=/bitnami/etcd
2022-08-21 12:42:47.411541 W | pkg/flags: unrecognized environment variable ETCD_ROOT_PASSWORD=
2022-08-21 12:42:47.411553 W | pkg/flags: unrecognized environment variable ETCD_CLUSTER_DOMAIN=mayastor-etcd-headless.mayastor.svc.cluster.local
2022-08-21 12:42:47.411563 W | pkg/flags: unrecognized environment variable ETCD_DISASTER_RECOVERY=no
2022-08-21 12:42:47.411639 W | pkg/flags: unrecognized environment variable ETCD_KEY_FILE=
2022-08-21 12:42:47.411659 W | pkg/flags: unrecognized environment variable ETCD_DAEMON_GROUP=etcd
2022-08-21 12:42:47.411665 W | pkg/flags: unrecognized environment variable ETCD_START_FROM_SNAPSHOT=no
2022-08-21 12:42:47.411670 W | pkg/flags: unrecognized environment variable ETCD_INIT_SNAPSHOT_FILENAME=
2022-08-21 12:42:47.411675 W | pkg/flags: unrecognized environment variable ETCD_INIT_SNAPSHOTS_DIR=/init-snapshot
2022-08-21 12:42:47.411681 W | pkg/flags: unrecognized environment variable ETCD_BASE_DIR=/opt/bitnami/etcd
2022-08-21 12:42:47.411687 W | pkg/flags: unrecognized environment variable ETCD_CERT_FILE=
2022-08-21 12:42:47.411692 W | pkg/flags: unrecognized environment variable ETCD_NEW_MEMBERS_ENV_FILE=/bitnami/etcd/data/new_member_envs
2022-08-21 12:42:47.411696 W | pkg/flags: unrecognized environment variable ETCD_DAEMON_USER=etcd
[WARNING] Deprecated '--logger=capnslog' flag is set; use '--logger=zap' flag instead
2022-08-21 12:42:47.411748 I | etcdmain: etcd Version: 3.4.15
2022-08-21 12:42:47.411755 I | etcdmain: Git SHA: aa7126864
2022-08-21 12:42:47.411759 I | etcdmain: Go Version: go1.12.17
2022-08-21 12:42:47.411763 I | etcdmain: Go OS/Arch: linux/amd64
2022-08-21 12:42:47.411768 I | etcdmain: setting maximum number of CPUs to 4, total number of available CPUs is 4
2022-08-21 12:42:47.411826 W | etcdmain: found invalid file/dir member_id under data dir /bitnami/etcd/data (Ignore this if you are upgrading etcd)
2022-08-21 12:42:47.411846 N | etcdmain: the server is already initialized as member before, starting as etcd member...
[WARNING] Deprecated '--logger=capnslog' flag is set; use '--logger=zap' flag instead
2022-08-21 12:42:47.412174 I | embed: name = mayastor-etcd-0
2022-08-21 12:42:47.412193 I | embed: data dir = /bitnami/etcd/data
2022-08-21 12:42:47.412206 I | embed: member dir = /bitnami/etcd/data/member
2022-08-21 12:42:47.412211 I | embed: heartbeat = 100ms
2022-08-21 12:42:47.412215 I | embed: election = 1000ms
2022-08-21 12:42:47.412219 I | embed: snapshot count = 100000
2022-08-21 12:42:47.412228 I | embed: advertise client URLs = http://mayastor-etcd-0.mayastor-etcd-headless.mayastor.svc.cluster.local:2379
2022-08-21 12:42:47.412234 I | embed: initial advertise peer URLs = http://mayastor-etcd-0.mayastor-etcd-headless.mayastor.svc.cluster.local:2380
2022-08-21 12:42:47.412241 I | embed: initial cluster = 
2022-08-21 12:42:47.435776 I | etcdserver: restarting member 482c4e3b4b340c15 in cluster ab0688bf84af917d at commit index 3430
raft2022/08/21 12:42:47 INFO: 482c4e3b4b340c15 switched to configuration voters=()
raft2022/08/21 12:42:47 INFO: 482c4e3b4b340c15 became follower at term 382
raft2022/08/21 12:42:47 INFO: newRaft 482c4e3b4b340c15 [peers: [], term: 382, commit: 3430, applied: 0, lastindex: 3430, lastterm: 6]
2022-08-21 12:42:47.437513 W | auth: simple token is not cryptographically signed
2022-08-21 12:42:47.445700 I | etcdserver: starting server... [version: 3.4.15, cluster version: to_be_decided]
raft2022/08/21 12:42:47 INFO: 482c4e3b4b340c15 switched to configuration voters=(5200617686277819413)
2022-08-21 12:42:47.446501 I | etcdserver/membership: added member 482c4e3b4b340c15 [http://mayastor-etcd-0.mayastor-etcd-headless.mayastor.svc.cluster.local:2380] to cluster ab0688bf84af917d
raft2022/08/21 12:42:47 INFO: 482c4e3b4b340c15 switched to configuration voters=(5200617686277819413 7753051432874073703)
2022-08-21 12:42:47.446691 I | etcdserver/membership: added member 6b985f3365cade67 [http://mayastor-etcd-1.mayastor-etcd-headless.mayastor.svc.cluster.local:2380] to cluster ab0688bf84af917d
2022-08-21 12:42:47.446725 I | rafthttp: starting peer 6b985f3365cade67...
2022-08-21 12:42:47.446789 I | rafthttp: started HTTP pipelining with peer 6b985f3365cade67
2022-08-21 12:42:47.448579 I | rafthttp: started streaming with peer 6b985f3365cade67 (writer)
2022-08-21 12:42:47.449896 I | rafthttp: started streaming with peer 6b985f3365cade67 (writer)
2022-08-21 12:42:47.450761 I | embed: listening for peers on [::]:2380
2022-08-21 12:42:47.451408 I | rafthttp: started peer 6b985f3365cade67
2022-08-21 12:42:47.451459 I | rafthttp: added peer 6b985f3365cade67
2022-08-21 12:42:47.451484 I | rafthttp: started streaming with peer 6b985f3365cade67 (stream MsgApp v2 reader)
raft2022/08/21 12:42:47 INFO: 482c4e3b4b340c15 switched to configuration voters=(5200617686277819413 7753051432874073703 10693245076485202313)
2022-08-21 12:42:47.451596 I | rafthttp: started streaming with peer 6b985f3365cade67 (stream Message reader)
2022-08-21 12:42:47.451678 I | etcdserver/membership: added member 946609bde8186189 [http://mayastor-etcd-2.mayastor-etcd-headless.mayastor.svc.cluster.local:2380] to cluster ab0688bf84af917d
2022-08-21 12:42:47.451707 I | rafthttp: starting peer 946609bde8186189...
2022-08-21 12:42:47.451730 I | rafthttp: started HTTP pipelining with peer 946609bde8186189
raft2022/08/21 12:42:47 INFO: raft.node: 482c4e3b4b340c15 elected leader 6b985f3365cade67 at term 382
2022-08-21 12:42:47.453692 I | rafthttp: started streaming with peer 946609bde8186189 (writer)
2022-08-21 12:42:47.456080 I | rafthttp: started streaming with peer 946609bde8186189 (writer)
2022-08-21 12:42:47.456119 I | rafthttp: peer 6b985f3365cade67 became active
2022-08-21 12:42:47.456166 I | rafthttp: established a TCP streaming connection with peer 6b985f3365cade67 (stream Message reader)
2022-08-21 12:42:47.456656 I | rafthttp: established a TCP streaming connection with peer 6b985f3365cade67 (stream MsgApp v2 reader)
2022-08-21 12:42:47.458271 I | rafthttp: started peer 946609bde8186189
2022-08-21 12:42:47.458357 I | rafthttp: added peer 946609bde8186189
2022-08-21 12:42:47.458393 I | rafthttp: started streaming with peer 946609bde8186189 (stream Message reader)
2022-08-21 12:42:47.458454 I | rafthttp: started streaming with peer 946609bde8186189 (stream MsgApp v2 reader)
2022-08-21 12:42:47.459051 N | etcdserver/membership: set the initial cluster version to 3.0
2022-08-21 12:42:47.459201 I | etcdserver/api: enabled capabilities for version 3.0
2022-08-21 12:42:47.459420 N | etcdserver/membership: updated the cluster version from 3.0 to 3.4
2022-08-21 12:42:47.459518 I | etcdserver/api: enabled capabilities for version 3.4
2022-08-21 12:42:47.460093 I | rafthttp: established a TCP streaming connection with peer 6b985f3365cade67 (stream MsgApp v2 writer)
2022-08-21 12:42:47.460368 I | rafthttp: established a TCP streaming connection with peer 6b985f3365cade67 (stream Message writer)
2022-08-21 12:42:47.461117 I | rafthttp: peer 946609bde8186189 became active
2022-08-21 12:42:47.461145 I | rafthttp: established a TCP streaming connection with peer 946609bde8186189 (stream Message reader)
2022-08-21 12:42:47.461244 I | rafthttp: established a TCP streaming connection with peer 946609bde8186189 (stream MsgApp v2 reader)
2022-08-21 12:42:47.463839 I | rafthttp: established a TCP streaming connection with peer 946609bde8186189 (stream MsgApp v2 writer)
raft2022/08/21 12:42:47 INFO: 482c4e3b4b340c15 switched to configuration voters=(7753051432874073703 10693245076485202313)
2022-08-21 12:42:47.467748 I | etcdserver/membership: removed member 482c4e3b4b340c15 from cluster ab0688bf84af917d
2022-08-21 12:42:47.468334 I | rafthttp: established a TCP streaming connection with peer 946609bde8186189 (stream Message writer)
2022-08-21 12:42:47.496241 I | etcdserver: 482c4e3b4b340c15 initialized peer connection; fast-forwarding 8 ticks (election ticks 10) with 2 active peer(s)
2022-08-21 12:42:48.468411 E | etcdserver: the member has been permanently removed from the cluster
2022-08-21 12:42:48.468448 I | etcdserver: the data-dir used by this member must be removed.
2022-08-21 12:42:48.468490 E | etcdserver: publish error: etcdserver: request cancelled
2022-08-21 12:42:48.468511 I | etcdserver: aborting publish because server is stopped
2022-08-21 12:42:48.468534 I | rafthttp: stopping peer 6b985f3365cade67...
2022-08-21 12:42:48.469542 I | rafthttp: closed the TCP streaming connection with peer 6b985f3365cade67 (stream MsgApp v2 writer)
2022-08-21 12:42:48.469566 I | rafthttp: stopped streaming with peer 6b985f3365cade67 (writer)
2022-08-21 12:42:48.470549 I | rafthttp: closed the TCP streaming connection with peer 6b985f3365cade67 (stream Message writer)
2022-08-21 12:42:48.470573 I | rafthttp: stopped streaming with peer 6b985f3365cade67 (writer)
2022-08-21 12:42:48.470601 I | rafthttp: stopped HTTP pipelining with peer 6b985f3365cade67
2022-08-21 12:42:48.470780 W | rafthttp: lost the TCP streaming connection with peer 6b985f3365cade67 (stream MsgApp v2 reader)
2022-08-21 12:42:48.470825 I | rafthttp: stopped streaming with peer 6b985f3365cade67 (stream MsgApp v2 reader)
2022-08-21 12:42:48.470954 W | rafthttp: lost the TCP streaming connection with peer 6b985f3365cade67 (stream Message reader)
2022-08-21 12:42:48.470982 E | rafthttp: failed to read 6b985f3365cade67 on stream Message (context canceled)
2022-08-21 12:42:48.470995 I | rafthttp: peer 6b985f3365cade67 became inactive (message send to peer failed)
2022-08-21 12:42:48.471025 I | rafthttp: stopped streaming with peer 6b985f3365cade67 (stream Message reader)
2022-08-21 12:42:48.471040 I | rafthttp: stopped peer 6b985f3365cade67
2022-08-21 12:42:48.471051 I | rafthttp: stopping peer 946609bde8186189...
2022-08-21 12:42:48.472216 I | rafthttp: closed the TCP streaming connection with peer 946609bde8186189 (stream MsgApp v2 writer)
2022-08-21 12:42:48.472248 I | rafthttp: stopped streaming with peer 946609bde8186189 (writer)
2022-08-21 12:42:48.474396 I | rafthttp: closed the TCP streaming connection with peer 946609bde8186189 (stream Message writer)
2022-08-21 12:42:48.474424 I | rafthttp: stopped streaming with peer 946609bde8186189 (writer)
2022-08-21 12:42:48.474463 I | rafthttp: stopped HTTP pipelining with peer 946609bde8186189
2022-08-21 12:42:48.474599 W | rafthttp: lost the TCP streaming connection with peer 946609bde8186189 (stream MsgApp v2 reader)
2022-08-21 12:42:48.474676 E | rafthttp: failed to read 946609bde8186189 on stream MsgApp v2 (context canceled)
2022-08-21 12:42:48.474739 I | rafthttp: peer 946609bde8186189 became inactive (message send to peer failed)
2022-08-21 12:42:48.474778 I | rafthttp: stopped streaming with peer 946609bde8186189 (stream MsgApp v2 reader)
2022-08-21 12:42:48.474902 W | rafthttp: lost the TCP streaming connection with peer 946609bde8186189 (stream Message reader)
2022-08-21 12:42:48.474931 I | rafthttp: stopped streaming with peer 946609bde8186189 (stream Message reader)
2022-08-21 12:42:48.474947 I | rafthttp: stopped peer 946609bde8186189
2022-08-21 12:42:48.475043 E | rafthttp: failed to find member 6b985f3365cade67 in cluster ab0688bf84af917d
2022-08-21 12:42:48.475245 E | rafthttp: failed to find member 946609bde8186189 in cluster ab0688bf84af917d
2022-08-21 12:42:48.475432 E | rafthttp: failed to find member 6b985f3365cade67 in cluster ab0688bf84af917d
2022-08-21 12:42:48.475823 E | rafthttp: failed to find member 946609bde8186189 in cluster ab0688bf84af917d
```

**Comment by wibed** [comment]
the solution is inproperly set hugepages.
it does fail upon reboot if hugepages are not present, which somehow on first install seem not to be much of an issue.

yet if statically provisioned the etcd cluster is reestablished after nodes inside it reboot.
